Resources/ConfigPresets/WizardsDen/leviathan.toml

```diff
@@ -4,7 +4,7 @@
 
 [game]
 hostname = "[EN] Wizard's Den Leviathan [US East]"
-soft_max_players = 80
+soft_max_players = 60
```

Comment:
I would increase Lizard's pop cap by 20 too, both so you get an easier comparison for round quality between the two and so that no one gets left out when the pop cap is already frequently hit. I don't think Miros or Spider would make up for that as they don't get consistent pop.